modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodel/v0.1.2.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/memodel/v0.1.2/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1"
+  ],
+  "shapes": [
+    {
+      "@id": "this:MEModelShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for...",
+      "targetClass": "nsg:MEModel",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1/shapes/ModelInstanceShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:eModel",
+              "name": "EModel",
+              "description": "EModel shape",
+              "editorialNote": "A node constraint should be added when it is supported: {{base}}/schemas/neurosciencegraph/simulation/emodel/v0.1.0/EModelShape",
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/simulation/emodel/v0.1.0/EModelShape",
+              "class": "nsg:EModel",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:morphology",
+              "name": "Morphology",
+              "description": "Morphology shape.",
+              "editorialNote": "A node constraint should be added when it is supported: {{base}}/schemas/neurosciencegraph/simulation/morphology/v0.1.1/shapes/MorphologyShape",
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/simulation/morphology/v0.1.1/shapes/MorphologyShape",
+              "class": "nsg:Morphology",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:mainScriptName",
```

Comment:
Why the isPartOf (towards the memodelrelease) is removed here ?

What is mainScriptName ? Is it the name of the model script ? If yes, I suggest to kee the modelscript as a file.